src/components/categoryitems.main.less

```diff
@@ -1,5 +1,101 @@
 @import './../style/common.less';
 
+.products-container {
+  position: relative;
+
+  .loading-product {
+    .category-items-listing {
+      position: relative;
+
+      &:before {
+        content: "";
+        height: 100%;
+        width: 100%;
+        background: rgba(255,255,255,0.8);
+        position: absolute;
+        top: 0;
+        left: 0;
+        z-index: 2;
+      }
+    }
+  }
+
+  .loader {
+    z-index: 999;
+    left: calc(50% - 60px);
+    top: 20%;
+    position: absolute;
+    border: 10px solid @complimentGrey;
+    border-top: 10px solid @complimentBlueColor;
+  }
+
+    .sort-product-menu {
+      position: absolute;
```

Comment:
If this is absolute, it will appear under the new components added to the layout:

![image](https://user-images.githubusercontent.com/39800563/57342544-608ede00-710d-11e9-888e-93842fc0f9d1.png)